src/AddTodoForm.js

```diff
@@ -1,14 +1,38 @@
 import React from 'react'
 
-const AddTodoForm = () => (
-    <div>
-        <form>
-            <label htmlFor="todoTitle">Title:</label>
-            <input id="todoTitle"></input>
-            <button>Add</button>
-        </form>
-    </div>
-)
+const AddTodoForm = ({ onAddTodo }) => {
+
+    const [todoTitle, setTodoTitle] = React.useState("");
+
+    const handleTitleChange = (event) => {
+        let newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+    };
+
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        //let todoTitle = event.target.title.value;
```

Comment:
It's better to just remove unused code.  The source control will have it if you need to reference it again.